barista-node/package.json

```diff
@@ -5,21 +5,18 @@
   "main": "main.js",
   "scripts": {
     "start": "node .",
-    "lint": "eslint ."
+    "lint": "eslint --ignore-path .gitignore . --fix"
```

Comment:
I would recommend against using `--fix` since in a CI environment,  you will never know that you had an issue, since it was fixed, but won't get commited back to the code

Reply:
Good point!